python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -967,7 +979,7 @@ def _add_objects(self, wcsf, inputCatalogRefs, sourceIndices, extensionInfo, col
         for inputCatalogRef in inputCatalogRefs:
             visit = inputCatalogRef.dataId['visit']
             inputCatalog = inputCatalogRef.get(parameters={'columns': columns})
-            detectors = set(inputCatalog['detector'])
+            detectors = np.unique(inputCatalog['detector'])
```

Comment:
Please add a comment that `np.unique` will output a sorted ordering of detector?